analytics-webapps/src/main/webapp/vue-app/common-components/components/common/SettingDatePicker.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <v-app id="analyticsDatePicker">
+    <v-menu
+      v-model="menu"
+      :close-on-content-click="false"
+      :nudge-right="40"
+      transition="scale-transition"
+      offset-y
+      min-width="auto">
+      <template v-slot:activator="{ on, attrs }">
+        <v-text-field
+          v-model="dateRangeText"
+          prepend-inner-icon="mdi-calendar"
+          class="pt-0 mt-0"
+          readonly
+          v-bind="attrs"
+          v-on="on" />
+      </template>
+      <v-date-picker
+        v-model="dates"
+        range
+        @input="selectedCustomDate()">
+        <div class="dateFooter footer">
+          <v-btn-toggle
+            class="d-flex flex-wrap justify-space-between"
+            tile
+            color="primary"
+            background-color="primary"
+            group>
+            <v-btn
+              v-for="(item, index) in periodOptions"
+              :key="index"
+              class="my-0"
+              :class="period && item.value === period.period && !useCustomDate? 'v-item--active v-btn--active' : ''"
```

Comment:
don't use `v-item--active v-btn--active` that are classes of other components and not CSS classes helpers